extensions/amp-story/1.0/amp-story-reaction-poll.css

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.i-amphtml-story-reaction-poll-container {
+  display: flex !important;
+  justify-content: center !important;
+  height: 90px !important;
+  max-width: 296px !important;
+  border-radius: 16px !important;
+  box-shadow: 0px 2px 4px #000000aa !important;
+  color: var(--reaction-accent-color) !important;
+  background: var(--reaction-background-color) !important;
+  overflow: hidden !important;
+
+  --post-select-scale: 0.5 !important;
+  --post-select-scale-variable: 1 !important;
+}
+
+.i-amphtml-story-reaction-poll-container[theme="dark"] {
+  --reaction-accent-color: white;
+}
+
+.i-amphtml-story-reaction-poll-container:not(.i-amphtml-story-reaction-post-selection) {
+  cursor: pointer !important;
+}
+
+.i-amphtml-story-reaction-option {
+  display: flex !important;
+  align-items: center !important;
+  justify-content: center !important;
+  text-align: center !important;
+  flex-direction: column !important;
+  flex-grow: 50 !important;
+  flex: 100 !important;
+  overflow: hidden !important;
+  position: relative;
+}
+
+.i-amphtml-story-reaction-poll-container:not(.i-amphtml-story-reaction-post-selection)
+  .i-amphtml-story-reaction-option:first-child::after {
+  content: '';
+  height: 100%;
+  position: absolute;
+  right: 1px;
+  width: 1px;
+  opacity: 0.3;
+  background-color: var(--reaction-accent-color);
+}
+
+.i-amphtml-story-reaction-post-selection
+  .i-amphtml-story-reaction-option.i-amphtml-story-reaction-option-selected::before {
+  content: '';
+  background-color: var(--reaction-accent-color);
+  opacity: 0.3;
+  position: absolute;
+  left: 0;
+  right: 0;
+  height: 100%;
+}
+
+.i-amphtml-story-reaction-option-title {
+  font-weight: 800 !important;
+  font-size: 28px !important;
+  margin: 0 !important;
+  line-height: 36px !important;
+  width: 124px !important;
+  height: 66px !important;
+  display: flex !important;
+  align-items: center !important;
+  justify-content: center !important;
+}
+
+.i-amphtml-story-reaction-post-selection
+  .i-amphtml-story-reaction-option-title {
+  transform: scale(var(--post-select-scale)) !important;
+  margin-bottom: -14px !important;
+  margin-top: -14px !important;
```

Comment:
Scaling inside a flex seemed to not update the width and height of the element, so the title was occupying the space as if it was still big. Removing 14px from each side makes it behave as if the size of the element is 1/2 of what it was, occupying 50% of the verical space. 

Found a stackoverflow that had the same problem:
https://stackoverflow.com/questions/50782231/flex-display-ignores-scale-transform-of-content

Anyways, alternatively I can change the height of the element and hope that when we add animations it will look good, so I did that and we can revise this when working on animations

Reply:
Also, I think animating the height could cause more redraws than animating the margins, so that's why I did it in the beginning, lmk if I should change it back to margins (vs currently changing height)